### ISSUE TITLE
HTML MIMEMultipart issue

### ISSUE THREAD

**Issue by dokime7** [theruziev/async_sender#229]
Hi again,

Sorry but I'm trying to increase at maximum the email deliverability :)

I can see on the code than the MIMEMultipart scafolding is well implemented but there is an issue for one case.
The case is when html is provided and no attachment.
In [this line](https://github.com/theruziev/async_sender/blob/66df33d20c138689a1646e536209b9fe4a73b60a/async_sender/api.py#L194) we are adding an 'alternative' part into a 'mixed' part but this 'mixed' part is not needed (only when attachments).
So Rspamd add the CTYPE_MIXED_BOGUS flag and decrease deliverability.

**Comment by theruziev** [comment]
Sorry but I didn't find any [spec](https://www.w3.org/Protocols/rfc1341/7_2_Multipart.html) for changing mimetype from alternative to mixed.

Can you describe more information about it.

**Comment by dokime7** [comment]
No, it's no changing alternative to mixed, but removing mixed part in case of there is no attachment.
It is not clear on the spec but there is an example with `Content-Type: multipart/alternative` at the root of email headers.

There is an exemple of scafolding [here](https://stackoverflow.com/questions/3902455/mail-multipart-alternative-vs-multipart-mixed) where attachments are present, but when not, it seems that the mixed part is not needed, so we start directly with alternative part.

We can also read [here](https://learn.microsoft.com/en-us/exchange/troubleshoot/administration/multipart-mixed-mime-message-format) that the mixed part is used only when attachments on Exchange Server.

Anyway, the purpose of this proposition is to inscrease deliverability by avoiding Rspamd bad score.
